.github/ISSUE_TEMPLATE/issue.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Issue
+about: Generic issue template for a proposed OpenAssetIO change
+title: Issue Title
+labels: ''
+assignees: ''
+
+---
+
+## Description
+
+## Acceptance Criteria
+
+- [ ] Requires release note updates.
+- [ ] Requires documentation updates.
+- [ ] Requires additional tests or test updates.
```

Comment:
[consider] Not sure we need reminding to add/update tests.

Reply:
I can see where you're coming from, as doing tests is more of a general rule. I thought this would be good just to remind ourselves to talk about the test requirements in planning, I felt sometimes we skim over specifically what needs tested, and whether existing tests already capture the modification or new ones need to be created.

Reply:
Just added a note, maybe checkboxes are more suited to the PR template as a reminder there?

Reply:
Perhaps these checkboxes should be in a different section, like "refinement discussion points" or something, to flag things that we should discuss when deciding on an implementation and/or scoping ACs.

Reply:
Had a think on this and changed my mind.

I wanted these to trigger neccesary detail-oriented discussion at planning, stuff that's easy to just nod and overlook, and I thought they had to be actual text on the issue.
However, I realise that in the case that a ticket has just been "thrown in", we'd have to open the edit pane in standup anyway, so we'd see comment formatted text. To that end, i've added them as a comment reminder, the same as everything else is.

[96524dd](https://github.com/OpenAssetIO/OpenAssetIO/pull/773/commits/96524ddb806b4be2530b03019b8d532471ebb19b)